projects/fsastorefrontstyles/scss/components/cms/_sales-illustration-main.scss

```diff
@@ -0,0 +1,26 @@
+%cx-fs-sales-illustration-main {
+  .sales-data,
+  .projections {
+    border: 1px solid var(--cx-color-3);
+    box-shadow: 0 8px 6px -6px #cccccc;
+    -moz-box-shadow: 0 8px 6px -6px #cccccc;
+    -webkit-box-shadow: 0 8px 6px -6px #cccccc;
+  }
+
+  .projections {
+    &-title {
+      background: var(--cx-color-4);
```

Comment:
Since we use this rule in many places, perhaps create separate class for this and reuse it?